### PR TITLE
Feat/en 2907 prometheus job name

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -51,8 +58,17 @@ func Start(elrondFacade MainApiHandler) error {
...
 
-func joinMonitoringSystem(port string, prometheusJoinURL string) error {
-	req, err := http.NewRequest("POST", prometheusJoinURL, strings.NewReader(port))
+func joinMonitoringSystem(elrondFacade MainApiHandler) error {
+	prometheusJoinUrl := elrondFacade.PrometheusJoinURL()
+	structToSend := prometheusDetails{
+		NodePort:  elrondFacade.RestApiPort(),
+		NetworkID: elrondFacade.PrometheusNetworkID(),
+	}
+	jsonValue, err := json.Marshal(structToSend)
```
Comment (sasurobert) [inline api/api.go]: add an empty line before.

File: api/api.go [hunk 2]
```diff
@@ -51,8 +58,17 @@ func Start(elrondFacade MainApiHandler) error {
...
+	if err != nil {
+		return err
+	}
+	req, err := http.NewRequest("POST", prometheusJoinUrl, bytes.NewBuffer(jsonValue))
```
Comment (sasurobert) [inline api/api.go]: add an empty line before. treat the error.

File: api/api.go [hunk 3]
```diff
@@ -23,11 +24,17 @@ type validatorInput struct {
 	Validator validator.Func
 }
 
+type prometheusDetails struct {
```
Comment (sasurobert) [inline api/api.go]: suggestion for simple prometheus.

File: api/api.go (hunk 1 above)
Comment (bogdan-rosianu) [reply api/api.go]: done

File: api/api.go (hunk 2 above)
Comment (bogdan-rosianu) [reply api/api.go]: fixed

File: api/api.go (hunk 3 above)
Comment (bogdan-rosianu) [reply api/api.go]: renamed

**Pull request merged**